### PR TITLE
Sitemap rel

### PR DESCRIPTION
**Pull request by nranas**
- Basic support for arrays. Will need more work when we have functionality in server.
- Support for sitemap rel.

### CODE REVIEW

File: src/api/factory.js [hunk 1]
```diff
@@ -9,6 +10,9 @@ export default function(data, parent) {
   else if (data.type == 'html') {
     return createHtmlContentProperty(data, parent)
   }
+  else if (data.type == 'array') {
```
Comment (hfjallemark) [inline src/api/factory.js]: No need for `else` anywhere here since we're returning, right?

File: src/components/sitemap.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+export default ({ executeAction, navigate, resource, update }) => {
+  if (!resource) {
+    return <div />
+  }
+
+  const Element = factory(resource)
+
+  return (
+    <div className={`ct-sitemap`}>
```
Comment (hfjallemark) [inline src/components/sitemap.js]: Use normal `'`.

File: src/api/factory.js (hunk 1 above)
Comment (nranas) [inline src/api/factory.js]: Right, fixed this now.

File: src/components/sitemap.js (hunk 2 above)
Comment (nranas) [inline src/components/sitemap.js]: Fixed

**Comment by nranas** [comment]
Solves #24

File: src/components/sitemap.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+export default ({ executeAction, navigate, resource, update }) => {
+  if (!resource) {
+    return <div />
+  }
+
+  const Element = factory(resource)
+
+  return (
+    <div className={'ct-sitemap'}>
```
Comment (hfjallemark) [inline src/components/sitemap.js]: No need for `{ }` for strings.

Comment (nranas) [inline src/components/sitemap.js]: ಠ_ಠ

**Pull request merged**